sessions/js-array-methods/for-each/js/index.js

```diff
@@ -1,39 +1,37 @@
 const colors = [
-	"#343434",
-	"#7FB5B5",
-	"#633A34",
-	"#354D73",
-	"#F3A505",
-	"#2A6478",
-	"#CC0605",
-	"#497E76",
-	"#D36E70",
-	"#FF2301",
-	"#4C514A",
-	"#2E3A23",
-	"#5D9B9B",
-	"#474A51",
-	"#EFA94A",
-	"#4D5645",
-	"#606E8C",
-	"#A5A5A5",
+	'#343434',
+	'#7FB5B5',
+	'#633A34',
+	'#354D73',
+	'#F3A505',
+	'#2A6478',
+	'#CC0605',
+	'#497E76',
+	'#D36E70',
+	'#FF2301',
+	'#4C514A',
+	'#2E3A23',
+	'#5D9B9B',
+	'#474A51',
+	'#EFA94A',
+	'#4D5645',
+	'#606E8C',
+	'#A5A5A5',
 ];
 
 /*
-1: The goal of this exercise is to create colored boxes from the array of hex codes. For that you will need to use the forEach method!
-  1.1: Create an anonymous array function inside the forEach function call which receives one input, namely the colorCode.
-  1.2: inside this arrow function do the following:
-    - create a new div with the document.createElement method.
-    - add the class 'color-box' to it.
-    - set the background color of the box by using the .style.backgroundColor property.
-    - append the box to the document.body.
+1: Create colored boxes from the array of hex codes.
+  1.1: Create an anonymous arrow function inside the forEach function call which receives one element, namely the colorCode.
+  1.2: inside this function create a new div and give it the color-box css class
+  1.3: Set the background-color of the new element to the colorCode.
+  1.4: Add the Element to the page.
```

Comment:
I think some sentences over complicate the start to this exercise. I am not too sure about the wordings `anonymous arrow function` and `function call which receives one element, namely the colorCode`. Why not simplify this by writing something like this:
```suggestion
1: Create colored boxes from the array of hex codes using the `forEach` array method.
  1.1: For each hex code a new `div` with the class `color-box` should be created
  1.2: The `background-color` of the `div` should be the current hex code
  1.3: The element should be added to the page
```
And a link to the [mdn article on `forEach`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach) could also be shared.

Reply:
I agree.

One of the main problems of our students is the complexity of the question/task.

The suggestions look good. 👍